api/v1/policyset_types.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2021 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// PolicySetSpec defines the desired state of PolicySet
+type PolicySetSpec struct {
+	Description string `json:"description,omitempty"`
+	// +kubebuilder:validation:Required
+	Policies []string `json:"policies"`
```

Comment:
Could you also please set `kubebuilder:validation:MinItems` so that the slice is never empty?